themes/digital.gov/src/scss/new/guides/_guide-landing.scss

```diff
@@ -0,0 +1,60 @@
+@use "uswds-core" as *;
+
+.dg-guides {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding-top: units(2);
+
+  @include at-media("tablet") {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  @include at-media("desktop") {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+.dg-guides-card {
+  border: 2px color("gray-cool-10") solid;
+  border-radius: units(1);
+  display: grid;
+  grid-template-rows: max-content 1fr;
+  margin: units(1);
+  padding: units(2);
+  position: relative;
+
+  &__kicker {
+    color: color("blue-70v");
+    font-weight: font-weight(200);
```

Comment:
Is this weight used anywhere else? It's more common to see `100`.

Keeping this consistent would allow us to save users from loading another font weight.